ip_localities.py

```diff
@@ -0,0 +1,79 @@
+from arcgis import GIS
+import pandas as pd
+import math
+
+
+def meters_to_long(x):
+    # Converts longitude point from Spherical Mercator EPSG:900913 to lat/lon in WGS84 Datum
+    origin_shift = 2 * math.pi * 6378137 / 2.0
```

Comment:
What's this number? I would recommend putting it in a variable so it's more obvious. Something like 
```
informative_name_for_misc_number = 6378137
origin_shift = math.pi * ...
```

Also, why 2 * pi * thing / 2? I think you can nix both 2s here...

Reply:
That is a constant used in converting the rather outdated long/lat system that Esri uses -- it won't be further modified hence why I didn't think it needed to be put in a variable. 

Would it be better in a variable?

Reply:
Variable isn't to change it, just for enhanced readability. Not as a parameter you pass in, just so it makes sense to the reader what it is.